homeassistant/components/marstek/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""The Marstek integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+
+from pymarstek import MarstekUDPClient, get_es_mode
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DEFAULT_UDP_PORT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
```

Comment:
Use the platforms const here:

https://github.com/home-assistant/core/blob/6a8106c0ebc688e6e3cc01401d626fda9b1a7471/homeassistant/components/abode/__init__.py#L48